### PR TITLE
Clear  react query caches after sending NFT

### PR DESCRIPTION
**Pull request by Jibz-Mysten**
## Description 
Clear  react query caches after sending NFT

https://user-images.githubusercontent.com/126525197/236018956-5f962bbf-ae96-4a02-adac-18555b130759.mov

### CODE REVIEW

File: apps/wallet/src/ui/app/pages/home/nfts/index.tsx [hunk 1]
```diff
@@ -14,7 +14,12 @@ import LoadingSpinner from '_components/loading/LoadingIndicator';
 import { NFTDisplayCard } from '_components/nft-display';
 import PageTitle from '_src/ui/app/shared/PageTitle';
 
-const MAX_FETCH_LIMIT = 50;
+//Sharing this with nft-transfer/TransferNFTForm
+//May need to move to a common file
+export const MAX_FETCH_LIMIT = 50;
+export const NFT_FILTER = {
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/pages/home/nfts/index.tsx:20]: Reuse this in the query key as well

File: apps/wallet/src/ui/app/pages/home/nft-transfer/TransferNFTForm.tsx [hunk 2]
```diff
@@ -48,7 +49,12 @@ export function TransferNFTForm({ objectId }: { objectId: string }) {
         },
         onSuccess: (response) => {
             queryClient.invalidateQueries(['object', objectId]);
-            queryClient.invalidateQueries(['objects-owned']);
+            queryClient.invalidateQueries([
```
Comment (williamrobertson13) [inline apps/wallet/src/ui/app/pages/home/nft-transfer/TransferNFTForm.tsx:52]: while this seems like the correct fix, I think manually plugging in the query keys used by `getOwnedObjects` here is super error-prone. Can we perhaps make a query key factory like https://tkdodo.eu/blog/effective-react-query-keys#use-query-key-factories instead?

**Review by williamrobertson13 — COMMENTED** [review]
If this change is urgent, I'm fine to give a stamp as long as we fix it as a fast-follow. Otherwise, I'd like to see us fix this in a way that's going to be less error prone down the road 😄

File: apps/wallet/src/ui/app/pages/home/nft-transfer/TransferNFTForm.tsx (hunk 2 above)
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/pages/home/nft-transfer/TransferNFTForm.tsx:52]: thinking if just clearing all `get-owned-objects` is better. We could send objects to another address of the same wallet as well

Comment (Jibz-Mysten) [reply apps/wallet/src/ui/app/pages/home/nft-transfer/TransferNFTForm.tsx]: You are right, I will add this to wallet hardening epic

**Review by pchrysochoidis — APPROVED** [review]

**Pull request merged**